lib/filterable_by.rb

```diff
@@ -74,6 +74,7 @@ def filter_by(hash = nil, **opts)
         return scope unless hash.respond_to?(:key?) && hash.respond_to?(:[])
 
         _filterable_by_config.each do |name, block|
+          block = ->(value, **) { where(name.to_sym => value) } if block.nil?
```

Comment:
Question out of curiosity - previous approach with populating "dummy" (well, default) `block` on init (on `filterable_by` call) seemed better 'cause it's done just once.

I'd not expect that approach to fail on abstract class (as block is not called, just remembered), why it's changed?

Reply:
Got it, now it runs in context of an actual (non-abstract class).